cachito/workers/nexus_scripts/rubygems_after_content_staged.groovy

```diff
@@ -72,16 +72,15 @@ void createRole(String name, String description, List<String> privileges) {
 
 // Main execution starts here
 request = new JsonSlurper().parseText(args)
-['rubygems_repository_name', 'raw_repository_name', 'password', 'username'].each { param ->
+['rubygems_repository_name', 'password', 'username'].each { param ->
```

Comment:
Just a suggestion for next time: it would have been nice to have one commit for changes to existing scripts and one for adding the new script

Reply:
Yes, makes sense.